tests/Bridge/Symfony/Messenger/DataPersisterTest.php

```diff
@@ -56,7 +57,9 @@ public function testPersist()
         $dummy = new Dummy();
 
         $messageBus = $this->prophesize(MessageBusInterface::class);
-        $messageBus->dispatch($dummy)->willReturn(new Envelope($dummy))->shouldBeCalled();
+        $messageBus->dispatch(Argument::that(function (Envelope $envelope) use ($dummy) {
+            return $dummy === $envelope->getMessage() && null !== $envelope->last(ContextStamp::class);
```

Comment:
Why is this change necessary?

Reply:
This changes consistent with code from testRemove().
By this changes we make sure that have envelop with ContextStamp.